tests/utils.py

```diff
@@ -35,6 +36,12 @@ def check_obj(
     assert_equals(obj_values, values)
 
 
+def convert_objects_to_str_in_json_serializable_object(
+    o: Union[list, dict]
+) -> Union[list, dict]:
+    return json.loads(json.dumps(o, default=str))
```

Comment:
Nice! Didn't know there is a `default` arg. Maybe this stuff I wrote earlier wasn't needed with this :) https://github.com/iterative/gto/blob/main/gto/utils.py#L24